Makefile

```diff
@@ -174,7 +174,12 @@ CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 ENVTEST ?= $(LOCALBIN)/setup-envtest
 
 ## Tool Versions
-KUSTOMIZE_VERSION ?= v3.8.7
+arch:= $(shell uname -m)
+ifeq ($(arch), ppc64le)
+ KUSTOMIZE_VERSION ?= v5.1.0
+else
+ KUSTOMIZE_VERSION ?= v3.8.7
```

Comment:
wouldn't be better for passing in `KUSTOMIZE_VERSION` as per use.
reason: 
1) these are all supported platforms
```
- amd64
  - arm64
  - s390x
  - ppc64le
```
2) unless  want to use `v5.1.0`, set it as default value wont help that much for others who wants run on  `ppc64le`

Reply:
For ppc64le only v5.1.0 is supported hence for ppc64le it is set to 5.1.0

Reply:
@ zdtsw, please let me know if any changes required

Reply:
Could we bump up KUSTOMIZE_VERSION for all arches?

Reply:
the old plan was to get all components + operator to work with new v5 all together, but the progess for that is bit too slow with low priority in the project. see tracker https://github.com/opendatahub-io/opendatahub-operator/issues/714 @manojnkumar 

another part for this PR did not get continue with is we have dropped adding new feature into v1 operator. 
if you still want this change only for ppc64le, could you make a new PR towards incubation branch ?  (assume you will use your own manifests for components) @SonalMahambrey1